index.js

```diff
@@ -3,13 +3,16 @@ var ansi = require("ansi-styles")
 ansi.line = { open: "\n", close: "" }
 ansi.tab = { open: "\t", close: "" }
 
-module.exports = (function clor (s) {
+module.exports = (function clor (s, open) {
   function $ (str) {
     return (str === undefined) ?
       $.toString() : clor(s + str + ansi.reset.close)
   }
 
-  $.toString = function () { return s + ansi.reset.close }
+  $.toString = function () {
```

Comment:
Hi @bucaran! `open` isn't declared anywhere: it's just an argument to the `clor()` function. I'm using it as a Boolean to flag that ansi.reset.close has not been added to the end of the String `s`.